src/main.c

```diff
@@ -501,13 +500,10 @@ int main(int argc, char **argv)
                 goto cleanup_no_lock;
         }
 
-        quiet = csv_mode || !no_html;
         self->output_file = output_file;
         self->cacert_file = cacert_file;
 
-        if (!csv_mode && self->output_file) {
-                quiet = false;
-        }
+        quiet = !self->output_file;
```

Comment:
While I do prefer this approach - it does break compatibility where we're being piped, and expected to be "quiet".